pkg/endpoint/policy.go

```diff
@@ -898,8 +898,6 @@ func (e *Endpoint) startRegenerationFailureHandler() {
 			}
 
 			regenMetadata := &regeneration.ExternalRegenerationMetadata{
-				// TODO (ianvernon) - is there a way we can plumb a parent
-				// context to a controller (e.g., endpoint.aliveCtx)?
 				ParentContext: ctx,
```

Comment:
This should probably just be `e.aliveCtx`; it seems awkward to tie regeneration to the controller's context.